projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-js/pdf-js-wrapper.ts

```diff
@@ -91,18 +94,32 @@ export class PdfJsWrapper {
     this.pdfViewer.eventBus.dispatch('findbarclose');
   }
 
+  public navigateTo(destination: any) {
+    if (destination instanceof Object) {
```

Comment:
this seems a bit weird, what else comes in?

Reply:
At certain times the destination is only a number that represents the location to navigate to, at others it is an object. I encountered this when testing on several different document types.

Reply:
maybe update the type info to reflect that, something like `Object | number` or `object | number` - they're different, I can't remember which is correct

Reply:
thanks for the comment. this has been updated.